docs/core/preview3/tools/using-ci-with-cli.md

```diff
@@ -74,12 +127,14 @@ The [travis-ci](https://travis-ci.org/) can be configured to install the .NET Co
 Just use:
 
 ```yaml
-dotnet: 1.0.0-preview2-003121
+dotnet: 1.0.0-rc4-0044771
 ```
 
 Travis can run both `osx` (OS X 10.11) and `linux` ( Ubuntu 14.04 ) job in a build matrix, see [example .travis.yml](https://github.com/dotnet/docs/blob/master/.travis.yml) 
```

Comment:
@mairaw @blackdwarf you can specify what osx you want to use (also do a build matrix on that) with normal travis conf, so let's use the default in these docs, and travis users can customize as usual ref [travis docs](https://docs.travis-ci.com/user/osx-ci-environment/#OS-X-Version)

A really good idea is also update directly the [f#, c#, vb official travis docs](https://docs.travis-ci.com/user/languages/csharp/) with .net core support (at least a link to this document)